installers/common.sh

```diff
@@ -208,6 +208,7 @@ function _setup_synchrona_service() {
     ExecStartPre=-/usr/bin/dtoverlay /boot/overlays/rpi-ad9545-hmc7044.dtbo
     ExecStart=/usr/local/bin/uvicorn --app-dir=$tmp/app/python/synchrona  main:app --host 0.0.0.0 --port 8000
     ExecStopPost=-/usr/bin/sh -c "/usr/bin/echo 6 > /sys/class/gpio/unexport"
+    ExecStartPost=-/usr/bin/bash /etc/raspap/synchrona/ad-synchrona-14-leds.sh > /dev/null 2>&1 &
```

Comment:
Did you make sure the script is killed when you do "systemctl stop synchrona"? At first glance, I don't think it is... Also, as a nit comment, add this line after `ExecStart`as this is part of the service startup.

At this point, I'm not sure if the LED script should not have it's own systemd service that could automatically start when one starts the synchrona service. I'm fairly sure this is doable but not 100% sure though...

Anyways, maybe just make sure you kill the script when the service is stopped for now...

Reply:
So I checked and the script gets killed on: "service synchrona stop"